bot/exts/evergreen/wolfram.py

```diff
@@ -54,7 +54,7 @@ async def send_embed(
     await ctx.send(embed=embed, file=f)
 
 
-def custom_cooldown(*ignore: int) -> Callable:
+def custom_cooldown(*ignore: list[int]) -> Callable:
```

Comment:
Just to be clear, what this actually means is that `ignore` is a tuple where each element is a list of integers. Is this your intention?

Reply:
This is not my intention. It will be changed to the proper annotation of `int`.